jwst/refpix/irs2_subtract_reference.py

```diff
@@ -387,6 +386,86 @@ def decode_mask(output, mask):
     return bits
 
 
+def rm_intermittent_badpix(data, scipix_n, refpix_r):
+    """Find pixels that are intermittently bad and replace with nearest
+    good value.
+
+    Parameters
+    ----------
+    data : 4-D ndarray
+        The data array in detector orientation.  This includes both the
+        science and interleaved reference pixel values.  `data` will be
+        modified in-place. The science data values will not be modified.
+
+    scipix_n : int
+        Number of regular (science) samples before stepping out to collect
+        reference samples.
+
+    refpix_r : int
+        Number of reference samples before stepping back in to collect
+        regular samples.
+
+    Returns
+    -------
+    data : 4-D ndarray
+        The data array in detector orientation.  This includes both the
+        science and interleaved reference pixel values.  The intermittently
+        bad pixels are now set to the nearest good reference pixel value.
+    """
+    # The intermittently bad pixels will be replaced for all integrations
+    # and all groups. The last group will be used to identify them
+    nits, ngroups, ny, nx = np.shape(data)
```

Comment:
I think "nits" here should be "nints", right?

Reply:
yep, I corrected it